collaboration-service/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "module",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.2",
+    "express": "^4.18.1",
+    "mongoose": "^6.6.1",
+    "socket.io": "^4.5.2",
+    "socket.io-client": "^4.5.2",
+    "uuid": "^9.0.0"
```

Comment:
Remove dependency uuid if not used. You can run `npm uninstall uuid` to remove it.